relay-server/Cargo.toml

```diff
@@ -109,6 +109,7 @@ symbolic-common = { version = "12.1.2", optional = true, default-features = fals
 symbolic-unreal = { version = "12.1.2", optional = true, default-features = false, features = [
     "serde",
 ] }
+sysinfo = "0.30"
```

Comment:
I reviewd the code path Relay will be using (memory), the crate is maintained by an author of the Rust foundation.